python/cudaq/kernel/ast_bridge.py

```diff
@@ -1469,6 +1469,24 @@ def bodyBuilder(iterVal):
                 opCtor([], [], [], [qubitA, qubitB])
                 return
 
+            if node.func.id == 'reset':
+                target = self.popValue()
+                # single qubit
+                if not quake.VeqType.isinstance(target.type):
```

Comment:
Is there a reason this isn't testing for a `RefType` explicitly? It has to be either a `RefType` or a `VeqType` at this point, eh? While `WireType` is allowed in the IR, I hope we don't want to allow that in the bridge.

Reply:
Thanks... I had considered only `RefType` and `VeqType` as the two mutually exclusive allowed types. 
Updated to explicitly check for these two types.